assets/prototype/domain/eventEditor/datesList/AddNewDateModal.js

```diff
@@ -0,0 +1,85 @@
+import classNames from "classnames";
+import { Classes, Overlay } from "@blueprintjs/core/lib/esm";
+// import { AppToaster } from "../EventEditor";
+import { Form, Field } from "react-final-form";
+import useCreateDateMutation from "../containers/mutations/useCreateDateMutation";
+
+const AddNewDateModal = ({ handleClose, isOpen }) => {
+	const [createDate, { data }] = useCreateDateMutation(CREATE_DATE);
+
+	const onSubmit = async ({ name }) => {
+		createDate({ variables: { input: name } });
+	};
+
+	const overlayProps = {
+		autoFocus: true,
+		canEscapeKeyClose: true,
+		canOutsideClickClose: true,
+		enforceFocus: true,
+		hasBackdrop: true,
+		usePortal: true,
+		useTallContent: false
+	};
+
+	const OVERLAY_EXAMPLE_CLASS = "docs-overlay-example-transition";
+	const OVERLAY_TALL_CLASS = "docs-overlay-example-tall";
```

Comment:
???

is this from the blueprint js code examples ???

I doubt these would be included in their production stylesheets

Reply:
yes, those are from examples, cleaned up a bit;
I didn't sweat too much about this modal, because as I understood from your PR we might not use `blueprint` at all: https://github.com/eventespresso/event-espresso-core/pull/1877#issue-343784674
***
and just be clear, even we will use that lib, we will have to do a cleaner usage of it's components with wrappers around that like I saw with `EspressoButton`
***
this was just a quick and dirty usage for the prototype